### DISCUSSION TITLE
Add nohup to scripts during install

### DISCUSSION THREAD

**Discussion by Node815** [tteck/Proxmox#3874] [Ideas]
I found myself getting distracted while waiting for a script to install tonight as my Home Assistant VM shutdown due to a very high system load (Not related to this script).   I reflexively switched to the machine to start it up and then realized I left the shell while it was doing it's thing.  

Having the **_[nohup](https://www.man7.org/linux/man-pages/man1/nohup.1.html)_**  command and piping the output to a file would be great so the user can then check for any error messages or the customary install information later.  When using that command, it doesn't kill the process if we do what I did!

**Comment by tteck** [comment]
Remember, each application utilizes five different scripts. When using `nohup`, the script runs in the background without interaction, meaning it can't accept user input, which would prevent further progress. All I can say is to stay focused. 😉
